spigot-tests/tests/acd-screen-share-test.ts

```diff
@@ -88,7 +88,7 @@ describe('ACD Screen Share via webrtc-sdk [acd-screenshare-sdk] [sdk] [stable]',
     logger.info('starting screenshare');
     await sdk.startScreenShare();
     sdk._streamingConnection.webrtcSessions.on('*', logger.log.bind(logger, 'sdk:webrtcSessions:event'));
-    logger.log('sdk._streamingConnection', sdk._streamingConnection);
+    // logger.debug('sdk._streamingConnection', sdk._streamingConnection);
```

Comment:
It was dumping the _entire_ streaming-client class in the console for each spigot test suite. It seemed a little excessive for the jenkins logs. I can add it back in if you think it would be useful.